apps/agora/server/project.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "agora-server",
```

Comment:
The convention, though undocumented, is to use the format `<product>-api` for the name of the API server project. Here this would be `agora-api`.

Reply:
Ah, is it because you want to use this name to the existing server and later create the project `agora-api` to start using the specification-first development workflow?